docs/docs/deploy/environment-variables.mdx

```diff
@@ -0,0 +1,131 @@
+| `RABBITMQ_SSL_CLIENT_CERTIFICATE` | The path to the SSL client certificate for the [`PikaEventBroker`](../production/event-brokers.mdx#pika-event-broker).                                                                                                |                   |
+| `RABBITMQ_SSL_CLIENT_KEY`         | The path to the SSL client key for the [`PikaEventBroker`](../production/event-brokers.mdx#pika-event-broker).                                                                                                        |                   |
+| `RASA_ENVIRONMENT`                | The Rasa environment for the [`PikaEventBroker`](../production/event-brokers.mdx#pika-event-broker) and the [`KafkaEventBroker`](../production/event-brokers.mdx#kafka-event-broker).                                 |                   |
+| `SECRET_MANAGER`                  | <RasaProLabel /> The type of [secret manager](../production/secrets-managers.mdx).                                                                                                                                    | `"vault"`         |
+| `TICKET_LOCK_LIFETIME`            | The lifetime of ticket locks, in seconds. It configures the [lock store](../production/lock-stores.mdx).                                                                                                              | `60`              |
+| `VAULT_URL`                       | <RasaProLabel /> The URl of the [HashiCorp Vault](../production/secrets-managers.mdx#hashicorp-vault-secrets-manager).                                                                                                |                   |
+| `VAULT_TOKEN`                     | <RasaProLabel /> The token to authenticate to the [HashiCorp Vault](../production/secrets-managers.mdx#hashicorp-vault-secrets-manager).                                                                              |                   |
+| `VAULT_NAMESPACE`                 | <RasaProLabel /> The namespace of the [HashiCorp Vault](../production/secrets-managers.mdx#hashicorp-vault-secrets-manager).                                                                                          |                   |
+| `VAULT_RASA_SECRETS_PATH`         | <RasaProLabel /> The path to Rasa secrets in the [HashiCorp Vault](../production/secrets-managers.mdx#hashicorp-vault-secrets-manager).                                                                               | `"rasa-secrets"`  |
+| `VAULT_TRANSIT_MOUNT_POINT`       | <RasaProLabel /> The mount point of the [HashiCorp Vault](../production/secrets-managers.mdx#hashicorp-vault-secrets-manager).                                                                                        |                   |
+
+### Model Storage
```

Comment:
maybe for the sake of completeness, we should still describe all the environment variables that are available here and documented elsewhere. I felt a bit lazy at first, but maybe it's better to have everything in the reference. Keen to get people's opinion 🤔 💡

Reply:
I think this is fine as we have the links here. I'd be against duplication of information as it'll make change management harder. The list on their respective pages (AWS, GCP, Azure) also gives it more context than is possible here